utf8proc.c

```diff
@@ -186,6 +186,37 @@ UTF8PROC_DLLEXPORT utf8proc_ssize_t utf8proc_encode_char(utf8proc_int32_t uc, ut
 }
 
 /* internal "unsafe" version that does not check whether uc is in range */
```

Comment:
Not sure I understand your question about test functions (may be total lack of sleep and jet lag...).  Could you explain further?  Thanks!